src/test/ProposalPayloadV2E2E.t.sol

```diff
@@ -0,0 +1,264 @@
+
+        ReserveConfigV2 memory expectedConfigPolygon = ReserveConfigV2({
+            symbol: "CRV",
+            underlying: CRV_POLYGON,
+            aToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal execution
+            variableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal exec
+            stableDebtToken: address(0), // Mock, no-validation because of the "dynamic" deployment on proposal execution
+            decimals: 18,
+            ltv: 2000,
+            liquidationThreshold: 4500,
+            liquidationBonus: 11000,
+            reserveFactor: 2000,
+            usageAsCollateralEnabled: true,
+            borrowingEnabled: true,
+            interestRateStrategy: NEW_INTEREST_RATE_STRATEGY_POLYGON,
+            stableBorrowRateEnabled: false,
+            isActive: true,
+            isFrozen: true
+        });
+
+        AaveV2Helpers._validateReserveConfig(expectedConfigPolygon, allConfigsPolygon);
+
+        AaveV2Helpers._validateInterestRateStrategy(
+            CRV_POLYGON,
+            ProposalPayloadPolygon(proposalPayloadPolygon).INTEREST_RATE_STRATEGY(),
+            InterestStrategyValuesV2({
+                excessUtilization: 30 * (AaveV2Helpers.RAY / 100),
+                optimalUtilization: 70 * (AaveV2Helpers.RAY / 100),
+                baseVariableBorrowRate: 3 * (AaveV2Helpers.RAY / 100),
+                stableRateSlope1: OLD_INTEREST_RATE_STRATEGY_POLYGON.stableRateSlope1(),
+                stableRateSlope2: OLD_INTEREST_RATE_STRATEGY_POLYGON.stableRateSlope2(),
+                variableRateSlope1: 14 * (AaveV2Helpers.RAY / 100),
+                variableRateSlope2: 300 * (AaveV2Helpers.RAY / 100)
+            }),
+            POLYGON
+        );
+
+        createConfigurationSnapshot("post-AaveV2Polygon-interestRateUpdate", AaveV2Polygon.POOL);
+    }
+
+    // Interest Strategy Ethereum
+
+    function testUtilizationAtZeroPercentEthereum() public {
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategy.calculateInterestRates(
+            CRV,
+            100 * 1e18,
+            0,
+            0,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At nothing borrowed, liquidity rate should be 0, variable rate should be 3% and stable rate should be 3%.
+        assertEq(liqRate, 0);
+        assertEq(stableRate, 3 * (AaveV2Helpers.RAY / 100));
+        assertEq(varRate, 3 * (AaveV2Helpers.RAY / 100));
+    }
+
+    function testUtilizationAtOneHundredPercentEthereum() public {
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategy.calculateInterestRates(
+            CRV,
+            0,
+            0,
+            100 * 1e18,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At max borrow rate, stable rate should be 313% and variable rate should be 317%.
```

Comment:
this is a bit weird.

while the test is correct, you essentially ensure **malfunction** of the IR.

StableRate should never be lower than variable rate - in this case it doesn't matter so much as stable rate is disabled. Still we consider it bad practice to design IR rates that will lead to unexpected behavior when stable was ever to be turned on.